src/test/java/com/artipie/rpm/misc/StorageLockerTest.java

```diff
@@ -37,6 +37,7 @@
  * Test {@link StorageLock}.
  * @since 0.9
  */
+@SuppressWarnings("PMD.AvoidDuplicateLiterals")
```

Comment:
@olenagerasimova if you follow my advice on the string literals below you will not need this suppression. :)